src/SslCertificate.php

```diff
@@ -31,6 +31,20 @@ public static function createForHostName(string $url, int $timeout = 30): self
         return Downloader::downloadCertificateFromUrl($url, $timeout);
     }
 
+    public static function createFromString(string $certificate): self
```

Comment:
Change the variable to `pathToCertificate`, that's a bit more clear.

Reply:
Added another method for this `createFromFile` so we have another option to fetch the certificate from a file instead of a string.